.travis.yml

```diff
@@ -12,10 +12,10 @@ compiler:
 matrix:
   include:
     - os: osx
-      osx_image: xcode8
+      osx_image: xcode8.3
       compiler: gcc
       env:
-        - MATRIX_EVAL="brew install gcc@6 && CC=gcc-6" # install and use gcc
+        - MATRIX_EVAL="rm -f '/usr/local/include/c++' && brew install gcc@6 && CC=gcc-6" # install and use gcc
```

Comment:
This looks  unsafe. Why did you remove this?

Reply:
This is suggested by brew.
Without this, brew terminates with an error saying there is already a link /usr/local/include/c++.
Please let me know if there is another good way other than this.

Reply:
But this was not happening in previous builds. If you really cannot remove it, please add a comment explaining why it is needed, possibly with a link to the suggestion from brew, so that we know where it comes from! It looks ugly! :)